nvflare/apis/utils/local_logger.py

```diff
@@ -15,25 +15,44 @@
 import logging
 import threading
 
-from nvflare.app_common.widgets.streaming import LogSender
-
 
 class LocalLogger:
+    handlers = None
     loggers = {}
     lock = threading.Lock()
 
+    @staticmethod
+    def initialize():
+        """        Initialize the LocalLogger to keep all the handlers before the adding of LogSender handler.
```

Comment:
I think it would be better to not include this "LogSender" in the docstring

Reply:
I expect you to modify the docstring when you move this into the log_streaming.py.

Reply:
no problem, let me do that in #135